tests/integration.rs

```diff
@@ -1,21 +1,58 @@
+use std::sync::atomic::AtomicU16;
 use std::thread;
 use std::time::Duration;
 
 use crate::utils::{get_redis_connection, start_redis_server_with_module};
 use anyhow::Context;
 use anyhow::Result;
-use redis::Value;
-use redis::{RedisError, RedisResult};
+use redis::{Connection, RedisError, RedisResult, Value};
+use redis_module::RedisValue;
+use utils::ChildGuard;
 
 mod utils;
 
+fn start_redis(module_name: &str, port: u16) -> Result<Vec<ChildGuard>, &'static str> {
+    Ok(vec![start_redis_server_with_module(module_name, port)
+        .map_err(|_| "failed to start redis server")?])
+}
+
+struct TestConnection {
+    _guards: Vec<ChildGuard>,
+    connection: Connection,
+}
+
+static mut TEST_PORT: AtomicU16 = AtomicU16::new(6479);
```

Comment:
Let use lazy static and avoid the unsafe?

Reply:
I am removing the `mut` and the `unsafe`, so this remains a piece of code without `lazy_static`.